Comp2025/src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -176,7 +153,8 @@ public LinearVelocity rangeProportional(LinearVelocity maxSpeed)
   public void SetCPUThrottleLevel(boolean throttle)
```

Comment:
This should also be restructured to take in the limelight name and operate on the limelight that is paased in. We want our methods to have similar APIs.